src/uwtools/cli/set_config.py

```diff
@@ -7,11 +7,12 @@
 from argparse import ArgumentError, ArgumentParser, HelpFormatter, Namespace
 from typing import List
 
-from uwtools import config
 from uwtools.exceptions import UWConfigError
 from uwtools.logging import setup_logging
 from uwtools.utils import cli_helpers
 
+from uwtools import config
```

Comment:
I get a lot of changes when I run `make format`. Could you run it in your clone and commit the changes? A `make format && make test` passes all tests for me, but reformats the imports in a bunch of files. For example, here, this import should be in a block with the other `uwtools` imports. But `make format` should take care of that.

Reply:
Aha - that's part of that issue I've been having where the `make format` keeps trying to break `test_logger`, there's clearly something wrong in my format definition

Reply:
I have `isort 5.12.0` and `black 23.3.0` so I'm not sure why its scrambling on me

Reply:
I have the same versions in my environment. We could look at this in a pairing session if you don't figure out what's up.

Reply:
Fixed - just had to clean something weird out of git and remake it